p11mod/p11mod.go

```diff
@@ -410,9 +410,29 @@ func (ll *llBackend) CreateObject(sh pkcs11.SessionHandle, temp []*pkcs11.Attrib
 }
 
 func (ll *llBackend) CopyObject(sh pkcs11.SessionHandle, o pkcs11.ObjectHandle, temp []*pkcs11.Attribute) (pkcs11.ObjectHandle, error) {
-	// TODO
-	log.Println("p11mod CopyObject: not implemented")
-	return 0, pkcs11.Error(pkcs11.CKR_FUNCTION_NOT_SUPPORTED)
+	session, err := ll.getSessionByHandle(sh)
+	if err != nil {
+		return 0, err
+	}
+
+	objectIndex := int(o - 1)
```

Comment:
Please add a comment explaining this line, similar to what `DestroyObject` has.

Reply:
Comment Added